src/liballoc/collections/btree/map.rs

```diff
@@ -1968,7 +1968,7 @@ where
                 (i, false) => i,
             },
             (_, Unbounded) => 0,
-            (true, Included(_)) => min_node.keys().len(),
```

Comment:
Did the tests hit this?

Reply:
I didn't try, I changed it before testing.

Reply:
I checked: none of these 3 calls to `keys()` triggers a debug assert. Maybe because it's theoretically impossible to be empty, probably because of missing unit tests.

Reply:
I've figured out the code enough to confirm that the two patterns with a `true` only occur after finding an element, so never on shared roots. And that the third one `(_, Unbounded)` indeed does occur on the shared root (if you write a test case for it).

But I give up on complete coverage: cannot get any test case to hit the `(true, Excluded(_))` pattern and I still don't really understand what the loop does.